packages/list-reusable-blocks/src/style.scss

```diff
@@ -6,10 +6,6 @@
 	align-items: center;
 	position: relative;
 	top: -3px;
-	// Todo: Make core button height and gutenberg button height equal.
-	.components-button {
-		height: 26px;
-	}
```

Comment:
Moved to a custom class name in `packages/list-reusable-blocks/src/components/import-dropdown/style.scss`.